app/components/footer.hbs

```diff
@@ -1,81 +1,129 @@
-<footer class='footer'>
-  <div class='footer__container'>
-    <div class='footer__sites-about'>
-      <div class='col'>
-        <h3 data-test-sites-title class='col__title'>Sites</h3>
-        <span data-test-underline class='col__underline'></span>
+{{#if this.isDev}}
+  <footer aria-label='footer under dev feature flag' class='footer-dev'>
```

Comment:
Why Event component not called from here i.e. <Event/>??

Reply:
Event component is now called from footer.